CheckerUI_xforms/CheckerUI/CheckerUI/CheckerUI/Helpers/Deparment/DeptView.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using CheckerUI.Enums;
+using CheckerUI.Helpers.DishesFolder;
+using CheckerUI.Models;
+using CheckerUI.ViewModels;
+using Lottie.Forms;
+using Xamarin.Forms;
+
+namespace CheckerUI.Helpers.Deparment
+{
+    public class DeptView : BaseViewModel
```

Comment:
imo a viewmodel should live in ViewModels

Reply:
But in this case it is not a direct part of the UI but rather it is used as a button, less correct to put it in the main folder because it is not main